src/code/z_sub_s.c

```diff
@@ -1186,13 +1186,21 @@ Actor* SubS_FindActor(PlayState* play, Actor* actorListStart, u8 actorCategory,
     return actor;
 }
 
-s32 SubS_FillLimbRotTables(PlayState* play, s16* limbRotTableY, s16* limbRotTableZ, s32 numLimbs) {
-    s32 i;
+/**
+ * Fills two tables with rotation angles that can be used to simulate idle animations.
+ *
+ * This is done for the desired limb by taking either the `sin` of the yTable value or the `cos` of th zTable value,
+ * multiplying by some scale factor (generally 200.0f), and adding that to the already existing rotation.
+ *
+ * Note: This effect is practically unnoticeable if the current animation already has motion involved.
+ */
+s32 SubS_FillFidgetTables(PlayState* play, s16* fidgetTableY, s16* fidgetTableZ, s32 tableLen) {
     u32 frames = play->gameplayFrames;
```

Comment:
It seems like this `fidgetTable` is expected to be updated every frame since the rots are updated internally because of `play->gameplayFrames`. The way it's phrased currently makes it seem like it would be an Init instead of an update. What about `SubS_UpdateFidgetTables`? Or at least mention it's an update somewhere in the description